docs/csharp/programming-guide/statements-expressions-operators/expression-bodied-members.md

```diff
@@ -0,0 +1,120 @@
+```
+
+where *expression* is any valid expression. 
+
+Support for expression body definitions was introduced for methods and property get accessors in C# 6 and was expanded in C# 7. Expression body definitions can be used with the type members listed in the following table. 
+
+|Member  |Supported as of... |
+|---------|---------|
+|[Method](#methods)  |C# 6 |
+|[Constructor](#constructors)   |C# 7 |
+|[Finalizer](#finalizers)     |C# 7 |
+|[Property Get](#property-get-statements)  |C# 6 |
+|[Property Set](#property-set-statements)  |C# 7 |
+|[Indexer](#indexers)       |C# 7 |
+
+## Methods
+
+An expression-bodied method consists of a single expression that returns a value whose type matches the method's return type, or, for methods that return `void`, that performs some operation. For example, types that override the <xref:System.Object.ToString</xref> method typically include a single expression that returns the string representation of the current object. 
+
+The following example defines a `Person` class that overrides the <xref:System.Object.ToString</xref> method with an expression body definition. It also defines a `Show` method that displays a name to the console. Note that the `return` keyword is not used in the `ToString` expression body definition.
+
+[!code-cs[expression-bodied-methods](../../../../samples/snippets/csharp/programming-guide/classes-and-structs/expr-bodied-methods.cs#1)]  
+
+For more information, see [Methods (C# Programming Guide)](../classes-and-structs/methods.md).
+ 
+## Constructors
+
+An expression body definition for a constructor typically consists of a single assignment statement or a method call that handles the constructor's arguments or initializes instance state. 
+
+The following example defines a `Location` class whose constructor has a single string parameter named *name*. The expression body definition assigns the argument to the `Name` property.
+
+[!code-cs[expression-bodied-constructor](../../../../samples/snippets/csharp/programming-guide/classes-and-structs/expr-bodied-ctor.cs#1)]  
+
+For more information, see [Constructors (C# Programming Guide)](../classes-and-structs/constructors.md).
+
+## Destructors
+
+An expression body definition for a destructor typically contains cleanup statements, such as statements that release unmanaged resources.
+
+The following example defines a destructor that uses an expression body definition to indicate that the destructor has been called.
+
+[!code-cs[expression-bodied-destructor](../../../../samples/snippets/csharp/programming-guide/classes-and-structs/expr-bodied-destructor.cs#1)]  
+
+For more information, see [Destructors (C# Programming Guide)](../classes-and-structs/destructors.md).
+
+## Property get statements
+
+If you choose to implement a property get accessor yourself, you can use an expression body definition for single-line statements that simply return the the proeprty value. Note that the `return` statement is not used.
```

Comment:
the the proeprty -> the property

should get be inline code here?

is not -> isn't